builder/actions/cmake.py

```diff
@@ -101,7 +101,7 @@ def _project_dirs(env, project):
     return source_dir, build_dir, install_dir
 
 
-def _build_project(env, project, cmake_extra, build_tests=False, args_transformer=None):
+def _build_project(env, project, cmake_extra, gcc_coverage=False, build_tests=False, args_transformer=None):
```

Comment:
I think we should always add new parameters at the end, so it doesn’t disturb legacy code.

Reply:
sure, but it's a private function, no one beside this package uses it.